### PR TITLE
Bug 1986426: Fixes for odf and ibm integration

### PR DESCRIPTION
**Pull request by afreen23**
- the value of `plural` should be `flashsystemclusters`
 - redirect to storage system list page
 - functions and constants need not be capitalized
 - removed inconsistency in naming wizard step files. All files to be suffixed with "step"
 - added right padding in the form
 - remove form tag inside form tags

Screenshot from 2021-07-26 19-04-58

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx [hunk 1]
```diff
@@ -19,6 +18,7 @@ import {
 import { ErrorHandler } from '../../error-handler';
 import { ExternalStorage } from '../../external-storage/types';
 import { NO_PROVISIONER } from '../../../../constants';
+import './backing-storage-step.scss';
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx:21]: renaming file

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/create-storage-class-step/create-storage-class-step.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.odf-create-storage-class__form {
+    padding-right: 10rem;
+}
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/create-storage-class-step/create-storage-class-step.scss:3]: added right padding for better look and feel

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/index.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 export { BackingStorage } from './backing-storage-step/backing-storage-step';
 export { CapacityAndNodes } from './capacity-and-nodes-step/capacity-and-nodes-step';
 export { ConnectionDetails } from './connection-details-step';
-export { CreateStorageClass } from './create-storage-class-step';
+export { CreateStorageClass } from './create-storage-class-step/create-storage-class-step';
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/index.ts:4]: file renaming

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/external-storage/ibm-flashsystem/index.tsx [hunk 4]
```diff
@@ -33,7 +32,7 @@ export const FlashSystemConnectionDetails: React.FC<ExternalComponentProps<Flash
   };
 
   return (
-    <Form>
+    <>
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/create-storage-system/external-storage/ibm-flashsystem/index.tsx:35]: form tag already used in parent component. Hence removing.

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/footer.tsx [hunk 5]
```diff
@@ -181,7 +183,13 @@ export const CreateStorageSystemFooter: React.FC<CreateStorageSystemFooterProps>
           type: 'wizard/setStepIdReached',
           payload: state.stepIdReached <= stepId ? stepId + 1 : state.stepIdReached,
         });
-        onNext();
+        stepName === StepsName(t)[Steps.ReviewAndCreate]
+          ? history.push(
+              `/k8s/ns/${CEPH_STORAGE_NAMESPACE}/${referenceForModel(
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/src/components/create-storage-system/footer.tsx:188]: We want to send there here or in the list page inside ODF?

Comment (afreen23) [reply frontend/packages/ceph-storage-plugin/src/components/create-storage-system/footer.tsx:188]: Once they are ready with dynamic plugin integration, we can change that.

**Pull request merged**